### PR TITLE
[ADD] Ticket Copy Method

### PR DESCRIPTION
**Pull request by DaniKL**
[ADD] Ticket copy method

[ADD] Contributors

### CODE REVIEW

File: helpdesk_mgmt/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -20,6 +20,9 @@
 * `Factor Libre <https://factorlibre.com>`_:
 
   * María Alhambra
+  * Daniel Cano
```
Comment (pedrobaeza) [inline helpdesk_mgmt/readme/CONTRIBUTORS.rst:23]: How can it be that adding a simple method supposes 3 new contributors?

File: helpdesk_mgmt/README.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ Helpdesk Management
     :target: https://runbot.odoo-community.org/runbot/282/11.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```
Comment (pedrobaeza) [inline helpdesk_mgmt/README.rst]: You are not generating the README through the scripts and touching manually, and thus this is being changed. Please restore this line as it was, use the tool for generating it, or let the bot to do it on merge.

File: helpdesk_mgmt/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (DaniKL) [reply helpdesk_mgmt/readme/CONTRIBUTORS.rst:23]: Because we did some work on the base but forgot to add ourselves as contributors.

Comment (pedrobaeza) [reply helpdesk_mgmt/readme/CONTRIBUTORS.rst:23]: Then please add separately in another PR and document that linking to the commits. I remember a commit from you doing the translation, which applies for the contributor role, but not from the others (but I haven't remembered in deep).

File: helpdesk_mgmt/README.rst (hunk 2 above)
Comment (DaniKL) [reply helpdesk_mgmt/README.rst]: I think it should be ok now, correct me if I'm wrong please.

File: helpdesk_mgmt/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (DaniKL) [reply helpdesk_mgmt/readme/CONTRIBUTORS.rst:23]: I did an early translation and mail templates along with Aitor. I will tell the rest to do a PR as you say.

**Comment by DaniKL** [comment]
I did some fixes, please let me know if there's anything wrong.

File: helpdesk_mgmt/models/helpdesk_ticket.py [hunk 3]
```diff
@@ -121,6 +121,18 @@ def create(self, vals):
...
+    @api.multi
+    def copy(self, default=None):
+        self.ensure_one()
+        if default is None:
+            default = {}
+        if "number" not in default:
+            default['number'] = self.env['ir.sequence'].next_by_code(
+                'helpdesk.ticket.sequence'
+            ) or '/'
+        res = super(HelpdeskTicket, self).copy(default)
```
Comment (aitorbouzas) [inline helpdesk_mgmt/models/helpdesk_ticket.py:133]: In Python3 you can call super with just `super()`.

**Review by aitorbouzas — APPROVED** [review]
LGTM, just made a comment to let you know, but it's ok this way.

**Comment by aitorbouzas** [comment]
Also, could you made a test for the copy method? It's too simple for not doing it.

**Comment by DaniKL** [comment]
I just did the test, please feel free to propose corrections.

**Comment by aitorbouzas** [comment]
Fine. What are your thoughts @pedrobaeza ?

**Comment by pedrobaeza** [comment]
@aitorbouzas if it's correct for you, approve the PR. I might be able to review it, but the general dynamic of the repo should be that Domatix and the rest of the initial contributors review this. for speeding up the merge, as I'm not able to review each of the PRs that are proposed to OCA.

**Review by pedrobaeza — APPROVED** [review]

**Review by carlos-domatix — APPROVED** [review]

**Pull request merged**